Gemfile.lock

```diff
@@ -405,69 +409,68 @@ GEM
       terminal-table (>= 1.4.0)
       thor (>= 0.16.0)
     jmespath (1.6.2)
-    json (2.6.3)
-    jsonpath (1.1.2)
+    json (2.7.1)
+    jsonpath (1.1.5)
       multi_json
     jwt (2.7.1)
     kubeclient (4.11.0)
       http (>= 3.0, < 6.0)
       jsonpath (~> 1.0)
       recursive-open-struct (~> 1.1, >= 1.1.1)
       rest-client (~> 2.0)
-    large_object_store (1.3.4)
-    llhttp-ffi (0.4.0)
+    large_object_store (1.7.0)
+      zstd-ruby (~> 1.5.5)
+    llhttp-ffi (0.5.0)
       ffi-compiler (~> 1.0)
       rake (~> 13.0)
-    lograge (0.12.0)
+    lograge (0.14.0)
       actionpack (>= 4)
       activesupport (>= 4)
       railties (>= 4)
       request_store (~> 1.0)
     logstash-event (1.2.02)
-    loofah (2.21.4)
+    loofah (2.22.0)
```

Comment:
### [loofah](https://github.com/flavorjones/loofah)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.21.4 → 2.22.0

[_[change-log](https://github.com/flavorjones/loofah/blob/main/CHANGELOG.md), [source-code](https://github.com/flavorjones/loofah), [gem-diff](https://my.diffend.io/gems/loofah/2.21.4/2.22.0)_]





<details>
<summary>Commits</summary>

A change of **6** commits. See the full changes on [the compare page](https://github.com/flavorjones/loofah/compare/v2.21.4...v2.22.0).

These are the individual commits:
- (09e11ad) [feat: adds :targetblank scrubber](https://github.com/flavorjones/loofah/commit/09e11adbdc568bbae1288a4a02ff425d752fad8a)
- (5345bb7) [Merge pull request #275 from hexdevs/add-target-blank-scrub](https://github.com/flavorjones/loofah/commit/5345bb7c9e2289c6cadf07300cb5f0f05d18439f)
- (4ad2e13) [Add noreferrer scrubber](https://github.com/flavorjones/loofah/commit/4ad2e135dbbe359b1311953838eb1ee171eb0d04)
- (c5cfb80) [Merge pull request #277 from wynksaiddestroy/feature/norefer…](https://github.com/flavorjones/loofah/commit/c5cfb807b9a4aaf723cb4222090a4d88a808cf51)
- (64e0a26) [update CHANGELOG](https://github.com/flavorjones/loofah/commit/64e0a26d46f015622b8059676a5badc077368737)
- (cb14ea7) [version bump to v2.22.0](https://github.com/flavorjones/loofah/commit/cb14ea7c03649504feb5491ea2e874fe13a4c353)


</details>